specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2020-12-01/examples/AzureIaasVm/BackupPolicies_List.json

```diff
@@ -0,0 +1,76 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "resourceGroupName": "SwaggerTestRg",
+    "vaultName": "NetSDKTestRsVault",
+    "api-version": "2019-06-15",
```

Comment:
please update the api-version in all examples

Reply:
Updated